lib/Raven/ReprSerializer.php

```diff
@@ -29,7 +29,11 @@ protected function serializeValue($value)
         } elseif (is_integer($value) || is_float($value)) {
             return (string) $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
-            return 'Object '.get_class($value);
+            $repr = 'Object ' . get_class($value);
+            if (method_exists($value, 'toSentry')) {
+                $repr .= ' ' . $value->toSentry();
```

Comment:
Choosing the return type of `toSentry` would be great instead of it always being casted to a string, would also prevent `array to string conversion` notices that are bound to happen with this approach.